### DISCUSSION TITLE
常用功能与过滤器(一)

### DISCUSSION THREAD

**Discussion by giscus** [oldwinter/knowledge-garden#19] [主发布站的评论系统]
# 常用功能与过滤器(一)

JSON入门

https://ob.hlcode.eu.org/%E5%B8%B8%E7%94%A8%E5%8A%9F%E8%83%BD%E4%B8%8E%E8%BF%87%E6%BB%A4%E5%99%A8%E4%B8%80

**Comment by Halecoder** [comment]
测试

**Reply by oldwinter** [reply]
哥们，那个script 去giscus自己生成一个。。。现在自动生成到我这里来了😂

**Reply by Halecoder** [reply]
找到了，我马上改

**Reply by Halecoder** [reply]
对不起了，还望大哥删掉，添麻烦了

**Comment by Halecoder** [comment]
我不知道，尴尬了，谢谢大哥提醒，我去看下


------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "oldwinter/knowledge-garden"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2022年9月10日(星期六) 晚上10:11
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [oldwinter/knowledge-garden] 常用功能与过滤器(一) (Discussion #19)





 
哥们，那个script 去giscus自己生成一个。。。现在自动生成到我这里来了😂
